docs/spec_draft.md

```diff
@@ -162,6 +162,7 @@ described below)
    * [logistic](#stablehlologistic)
```

Comment:
Similarly to the nearby PR, please take a look the [checklist](https://github.com/openxla/stablehlo/pull/260) and go through its items. In particular, you'll need to update status.md and StablehloOps.td.